source/wp-content/themes/wporg-developer/scss/prism.scss

```diff
@@ -222,6 +222,10 @@ pre.line-numbers > code {
 	border-top-left-radius: 0.3em;
 	border-top-right-radius: 0.3em;
 
+	@media screen and (max-width: 876px) {
+		top: 0;
```

Comment:
This is adjusted after rebasing the trunk. See https://github.com/WordPress/wporg-developer/pull/125/commits/f43a9e62989a1056179f0fb9011eae3ca1393ed1.